test/e2e/tracks.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Tracks Page E2E', () => {
+  test.beforeEach(async ({ page }) => {
+    page.on('console', (msg) => {
+      if (msg.type() === 'error') {
+        console.log('Console error:', msg.text());
+      }
+    });
+    page.on('pageerror', (error) => {
+      console.log('Page error:', error.message);
+    });
+    await page.goto('/tracks');
+    await page.waitForLoadState('networkidle');
+    await page.waitForSelector('body', { timeout: 10000 });
+    await page.screenshot({ path: 'debug-page.png' });
```

Comment:
I don't think that screenshot test is a good idea here, because if any of the tracks changes, or a new one is added it will fail. I suppose it could be fixed by making a separate backend for tests where the data is always the same. You could remove this or leave it here, but just keep in mind that it will potentially fail in the future.